### DISCUSSION TITLE
Shows emojis that doesn't exist in Twemoji

### DISCUSSION THREAD

**Discussion by TikoyX** [ShahriarKh/twemoji-cheatsheet#18] [Bugs]
First thank you for a nice website.

However, i ran into a problem.
The website claims the "eye in speech bubble" is part of Twemoji. But that emoji does not exist in the twemoji-repo.
Is there something i have overlooked here?

**Comment by ShahriarKh** [comment]
Thank you TikoyX!
Eye in speech bubble exists in Twemoji:
![](https://cdn.jsdelivr.net/gh/twitter/twemoji@14.0.2/assets/72x72/1f441-200d-1f5e8.png)
All of the emojis are retrieved automatically, and I didn't add anything manually, so if an emoji is shown then Twemoji supports it. Can you give me a link to where you looked? 🤔

**Reply by TikoyX** [reply]
Okay. I was looking at the official twemoji repository and couldn't find the emoji in there. 
I therefore assumed it was not part of it or is there something i am misunderstanding? 🤔 

https://github.com/twitter/twemoji/tree/master/assets/svg

**Reply by ShahriarKh** [reply]
Well, maybe the emoji isn't there because of this:
> "Sorry, we had to truncate this directory to 1,000 files. 2,689 entries were omitted from the list."

**Reply by TikoyX** [reply]
Nope. Downloaded them all, still couldn't find it

**Comment by TikoyX** [comment]
Updated list of emojis that the cheatsheet claims is part of twemoji but isnt:

- 00a9
- 00ae
- 002a-fe0f-20e3
- 0023-fe0f-20e3
- 0030-fe0f-20e3
- 0031-fe0f-20e3
- 0032-fe0f-20e3
- 0033-fe0f-20e3
- 0034-fe0f-20e3
- 0035-fe0f-20e3
- 0037-fe0f-20e3
- 0036-fe0f-20e3
- 0038-fe0f-20e3
- 0039-fe0f-20e3
